weave/trace_server/clickhouse_trace_server_batched.py

```diff
@@ -602,7 +574,99 @@ def _expand_call_refs(
                             val["_ref"] = ref.uri()
                         set_nested_key(calls[i], col, val)
 
-    @ddtrace.tracer.wrap(name="clickhouse_trace_server_batched.calls_delete")
+    @ddtrace.tracer.wrap(name="clickhouse_trace_server_batched.calls_descendants")
+    def calls_descendants(
```

Comment:
Should this be a new endpoint or a new way to query the existing `/calls/stream_query` endpoint. For example with a `ancestor_ids` argument?

I presume this would de-dup some of the logic below?

Reply:
could totally be rolled into that, this is how I thought about the current structure:

Pros: 
- doesn't increase complexity on our single most important endpoint, which is already overloaded imo.
- easier to change in the future if we want to go a different route 
- we don't have to support stuff like storage or other flags that might be impossibly hard to compute but are on the streaming endpoint.

Cons:
- some duplicate code
- another endpoint, which is specialized and not very flexible